docs/zh/latest/plugins/grpc-transcode.md

```diff
@@ -23,20 +29,40 @@ title: grpc-transcode
 
 ## 描述
 
-HTTP(s) -> APISIX -> gRPC server
+`grpc-transcode` 插件可以在 HTTP 和 gRPC 请求之间进行转换。
 
-## Proto
+APISIX 接收 HTTP 请求后，首先对请求进行转码，并将转码后的请求转发到 gRPC 服务，获取响应并以 HTTP 格式将其返回给客户端。
 
-### 参数
+<!-- TODO: use an image here to explain the concept better -->
```

Comment:
you can use picture in this post:https://apisix.apache.org/zh/blog/2022/01/25/apisix-grpc-web-integration